### ISSUE TITLE
metroSwitchSetup option missing?

### ISSUE THREAD

**Issue by TorakikiSan** [olton/Metro-UI-CSS#1768]
I Olton!

I am trying to use the setup options for various components.

I don't find the equivalent option for data-on and data-off in the switch component.

I try with

```
metroSwitchSetup={
                  onCaption: "Yes",
                  offCaption: "No"
                 };

```
or

```
metroSwitchSetup={
                  on: "Yes",
                  off: "No"
                 };


```
with no success.

Any hint?

**Comment by olton** [comment]
Fixed in 4.5.0

+ [x] Switch: To show on-off label you must set attribute `data-show-on-off='true'` and use attributes: `data-on`, `data-text-on`, `data-off`, `data-text-off` or `i18n` configs with attribute `data-locale`
+ [x] Switch: added `i18n` support with props `locale.switch.on`, `locale.switch.off`
+ [x] Switch: added changing on-off text in runtime with API method `i18n(locale_name)`

```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
    <link href="../metro/css/metro-all.css?ver=@@b-version" rel="stylesheet">

    <title>Test Switch - Metro 4 :: Popular HTML, CSS and JS library</title>

</head>
<body class="m4-cloak">
<div class="container">
    <h1>Switch test page</h1>

    <div class="mb-10">
        <button class="button">en-US</button>
        <button class="button">uk-UA</button>
        <button class="button">ru-RU</button>
    </div>
    <div>
        <input type="checkbox" data-role="switch" data-on="on" data-off="off">
        <input type="checkbox" data-role="switch" data-text-on="yes" data-text-off="no">
        <input type="checkbox" data-role="switch" checked data-locale="uk-UA">
        <input type="checkbox" data-role="switch" id="switch1">
    </div>
</div>

<script>
    window.metroSwitchSetup = {
        showOnOff: true
    }
</script>
<script src="../metro/js/metro.js?ver=@@b-version"></script>
<script>
    $("button").on("click", function(){
        Metro.getPlugin("#switch1", "switch").i18n($(this).text())
    })
</script>
</body>
</html>
```

**Comment by TorakikiSan** [comment]
Many thank's!
